packages/suite-build/tsconfig.json

```diff
@@ -2,10 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
-        "allowJs": true,
-        "esModuleInterop": true,
-        "resolveJsonModule": true,
-        "baseUrl": "."
-    }
+        "outDir": "./libDev"
+    },
+    "include": ["**/*.ts", "**/*.tsx", "**/*.js"],
```

Comment:
there shoudnt be any tsx files in this package (no react here)

Reply:
simplified to just `"."`